app/controllers/finders_controller.rb

```diff
@@ -353,4 +359,8 @@ def filter_query_array(arr)
       arr.reject { |v| v == "all" }.compact.presence
     end
   end
+
+  def page_under_test?
```

Comment:
I think this can be moved to the concern?

Reply:
done!